src/constants/wildbookSystemGuids.js

```diff
@@ -0,0 +1,5 @@
+const wildbookSystemGuids = {
+  'e144fec3-2f47-4195-8025-a8915f9a52a1': 'Wildbook for Zebras',
```

Comment:
This is the system GUID for zebra.wildme.org. It is  a site-setting that is a unique guid per instance.

Reply:
would this be a `wildbookSystemGuid` or a `codexSystemGuid`?

Reply:
It is a codex system GUID. What do you think of changing the name to `wildbooksBySystemGuids` then?

If we thought we were going to store other information too, I could change it too something like
```
const systemInformation = {
  'e144fec3-2f47-4195-8025-a8915f9a52a1': {
    migratedFrom: 'Wildbook for Zebras',
    // ... other information
  }
};
```

Perhaps this isn't even data that the front end should be storing, but something that we should request from houston.

Reply:
Renamed to `wildbookBySystemGuid` in https://github.com/WildMeOrg/codex-frontend/pull/386/commits/f346ab342487247e98ac4b0799299d54cd94bbaa